src/Domain/Insights/InsightFactory.php

```diff
@@ -153,10 +161,10 @@ private function getSniffKey(Error $error): string
      *
      * @return \Symplify\EasyCodingStandard\Error\ErrorAndDiffCollector
      */
-    private function getSniffCollector(array $config): ErrorAndDiffCollector
+    private function getCollector(array $config): ErrorAndDiffCollector
```

Comment:
One of us is getting a lot of cleanup with this method depending on if phpstan or phpcs is getting merged in first ;)

Reply:
Yeah, I'm sorry :sweat_smile: I may have had to start from your branch

Reply:
It's fine. I think both of us have done some valuable work to improve this part here, let's hope we can make a nice merge when we get to that :)